libs/features/mas/web-components/src/variants/mini-compare-chart.css.js

```diff
@@ -68,6 +68,14 @@ export const CSS = `
     height: var(--consonant-merch-card-mini-compare-chart-icon-size);
   }
 
+  merch-card[variant="mini-compare-chart"] .footer-rows-title {
+    font-color: #2C2C2C;
+    font-weight: 700;
```

Comment:
do we have merch variables for these values?

Reply:
I could not find variables for these 2. For others using variables

Reply:
I think it is: `--merch-color-grey-80`

Reply:
done